latest/index.bs

```diff
@@ -175,7 +175,261 @@ For this example we assume an image with 5 dimensions and axes called `t,c,z,y,x
+        │       │   |   ...
+        │       |   └── n
+        |       |
+        |       ├── indices   # You MUST add a one-dimensional zarr array named "indices".
+        |       |   |         # `indices` MAY be chunked as the user desires.
+        |       |   ├── .zarray
+        |       |   |
+        |       |   ├── 0
+        │       │   |   ...
+        │       |   └── n
+        |       |
+        |       └── indptr    # You MUST add a one-dimensional zarr array named "indptr".
+        |           |         # `indptr` MAY be chunked as the user desires.
+        |           ├── .zarray
+        |           |
+        |           ├── 0
+        │           |   ...
+        │           └── n
+        |
+        ├── varp              # You MAY add a varp group comtainer. The varp group contains sparse arrays that annotate the columns in X.
+        |   │                 # The rows in each array MUST be index-matched to the columns in X (if present). 
+        |   |
+        │   ├── .zgroup
+        |   |
+        |   ├── .zattrs       # `.zattrs` MAY contain `"encoding-type"`, which is set to `"dict"` by AnnData.           
+        |   │                 # `.zattrs` MAY contain `"encoding-version"`, which is set to `"0.1.0"` by AnnData.
+        |   │
+        │   └── varp_0        # You MAY add a zarr group for each varp array.
+        |       |             # Each varp array MUST have the same number of rows as columns in X.
+        |       |
+        │       ├── .zgroup
+        |       |
+        |       ├── .zattrs   # `.zattrs` MAY contain `"encoding-type"`, which is set to `"csr_matrix"` by AnnData.           
+        |       │             # `.zattrs` MAY contain `"encoding-version"`, which is set to `"0.1.0"` by AnnData.
+        |       |             # `.zattrs` MAY contain `"shape"` which is an array giving the shape of the densified array.
+        |       |
+        |       ├── data      # You MUST add a one-dimensional zarr array named "data". 
+        |       |   |         # `data` MAY be chunked as the user desires.
+        |       |   ├── .zarray
+        |       |   |
+        |       |   ├── 0
+        │       │   |   ...
+        │       |   └── n
+        |       |
+        |       ├── indices   # You MUST add a one-dimensional zarr array named "indices".
+        |       |   |         # `indices` MAY be chunked as the user desires.
+        |       |   ├── .zarray
+        |       |   |
+        |       |   ├── 0
+        │       │   |   ...
+        │       |   └── n
+        |       |
+        |       └── indptr    # You MUST add a one-dimensional zarr array named "indptr".
+        |           |         # `indptr` MAY be chunked as the user desires.
+        |           ├── .zarray
+        |           |
+        |           ├── 0
+        │           |   ...
+        │           └── n
+        |
+        └── uns               # You MAY add a uns containter to store unstructured data.
```

Comment:
@kevinyamauchi one missing datatype: structured numpy arrays.

Tbh, they're not great to work with, but our differential expression code still uses them.

Should we include them?

Reply:
structured in the sense of the object codec? (https://zarr.readthedocs.io/en/stable/tutorial.html#object-arrays) I'd be hesitant until that makes its way into the Zarr spec.

Reply:
Not object codecs, structured as in structured numpy dtypes.